Service/CallService.php

```diff
@@ -57,13 +57,15 @@ public function __construct(AuthenticationService $authenticationService, Entity
     public function getCertificate(array $config): array
     {
         $configs = [];
-        if (isset($config['cert'])) {
-            $configs['cert'] = $this->fileService->writeFile('certificate', $config['cert']);
+        if (isset($config['cert']) === true) {
+            $contents = is_array($config['cert']) ? $config['cert'][0] : $config['cert'];
+            $configs['cert'] = $this->fileService->writeFile('certificate', $contents);
         }
-        if (isset($config['ssl_key'])) {
-            $configs['ssl_key'] = $this->fileService->writeFile('privateKey', $config['ssl_key']);
+        if (isset($config['ssl_key']) === true) {
+            $contents = is_array($config['ssl_key']) ? $config['ssl_key'][0] : $config['ssl_key'];
```

Comment:
Zit dit altijd in array [0] en kan het niet dat het direct de cert is als string?

Reply:
Als dat het geval is dan word de string gebruikt ipv array [0]

`: $config['ssl_key'];`

Reply:
Correct @WilcoLouwerse